README.md

```diff
@@ -42,6 +42,21 @@ conda env create -f environment.yml
 conda activate lsa
 ```
 
+#### Docker Installation
+
+first buid the image:
+
+```
+docker build --tag 'lang-segment-anything' .
+```
+
+then run it (on windows use powershell):
+
+```
+docker run --name running-langsam --mount type=bind,source=D:\data,target=/workspace/data --gpus all -it 'lang-segment-anything' 
```

Comment:
Not sure how why we are suggesting bind mount data here as it is not required to run this project.